src/Uno.UI/UI/Xaml/Window.Android.cs

```diff
@@ -76,6 +78,34 @@ private static Window InternalGetCurrentWindow()
 
 		internal void RaiseNativeSizeChanged()
 		{
+#if __ANDROID_30__
```

Comment:
Will these APIs work on devices running Android 29 and below?

Reply:
They are documented to be available starting with API 30, and apparently we didn't had that issue in previous versions so fallbacking on the old code seems legit.

Reply:
Yep, that's the reason why the tests was failing on the CI for Android 🤦‍♂️